### PR TITLE
HRSPLT-224 Fixes HRS Time Entry Failures

### PR DESCRIPTION
**Pull request by vertein**
Before:
image

After:
image

### CODE REVIEW

**Comment by jhanstra** [comment]
:+1:

File: hrs-portlets-webapp/src/main/java/edu/wisc/portlet/hrs/web/timeabs/TimeSheetDataController.java [hunk 1]
```diff
@@ -48,15 +51,32 @@ public void setTimeSheetDao(TimeSheetDao timeSheetDao) {
...
     @ResourceMapping("timeSheets")
     public String getTimeSheets(ModelMap modelMap) {
         final String emplid = PrimaryAttributeUtils.getPrimaryId();
 
         final List<TimeSheet> timeSheets = this.timeSheetDao.getTimeSheets(emplid);
-
-        modelMap.addAttribute("report", timeSheets);
         
+        //sorts the list from most recent time sheets to oldest time sheets
+        Collections.sort(timeSheets, new Comparator<TimeSheet>(){
```
Comment (apetro) [inline hrs-portlets-webapp/src/main/java/edu/wisc/portlet/hrs/web/timeabs/TimeSheetDataController.java:67]: I'd have implemented an external `TimeSheetComparator` class so as to unit test it, but it'd be a trivial unit test and I accept that some developers prefer this style of code.

**Comment by timlevett** [comment]
:+1:

File: hrs-portlets-webapp/src/main/java/edu/wisc/portlet/hrs/web/timeabs/TimeSheetDataController.java [hunk 2]
```diff
@@ -48,15 +51,32 @@ public void setTimeSheetDao(TimeSheetDao timeSheetDao) {
     }
 
     
+    /**
+     * Gets Time sheets - limited to most recent 80 results
+     * @param modelMap
+     * @return the most recent timesheets, limited to 80
+     */
```
Comment (apetro) [inline hrs-portlets-webapp/src/main/java/edu/wisc/portlet/hrs/web/timeabs/TimeSheetDataController.java:58]: Nicely done on the JavaDoc.

Maybe "Gets most recent up to 80 TimeSheets for the requesting user."

**Comment by apetro** [comment]
:+1:

**Comment by vertein** [comment]
Merged after changing the JavaDoc per a suggestion.  Didn't do the separate class idea.  Andrew's reasoning was 100% my thought process, I started to down that road for unit tests, then backed out.

**Pull request merged**